Haskell/tests/syntax_test_haskell.hs

```diff
@@ -2455,7 +2455,7 @@
 
     {- Class Method Declarations -}
     class TyCls a where
-      nethod1 :: a -> Bool
+      method0 :: a -> Bool
```

Comment:
This was a bit odd... Normally I'd use `_` as a suffix, but there's already a numeric suffix and there's already a `method1` below...

Reply:
It's also possible `method1` would be ok. I didn't have the energy to figure out if this was _intentionally_ different from `method1` or just _accidentally_ different...

Reply:
We actually don't care about duplicates in syntax tests, so both `method0` and `method1` are ok.